src/leaf-model-types.ts

```diff
@@ -23,11 +23,11 @@ export interface IModelOutputState {
   habitatFeatures: Set<HabitatFeatureType>;
   chemistryTestResults: ChemistryTestResult[];
 }
-export interface IModelTransientState {
+export interface ILeafModelTransientState {
   time: number;
 }
 export interface ISerializableTrayObject extends Omit<TrayObject, "image" | "dragImage" | "selectionPath" | "width" | "height"> {}
-export interface ISerializableModelOutputState extends Omit<IModelOutputState, "trayObjects" | "habitatFeatures">  {
+export interface ISerializableModelOutputState extends Omit<ILeafModelOutputState, "trayObjects" | "habitatFeatures">  {
```

Comment:
Should this be `ISerializableLeafModelOutputState` for consistency?

Reply:
Good call -- I'll make that change.